build.rs

```diff
@@ -121,7 +145,10 @@ fn build(frameworks_path: &str) {
     // Begin building the bindgen params.
     let mut builder = bindgen::Builder::default();
 
-    builder = builder.clang_arg(format!("-F/{}", frameworks_path));
+    builder = builder
+        .clang_arg(format!("-F/{}", frameworks_path))
+        .clang_arg("-isysroot")
```

Comment:
Has this been tested on older versions of osx ?

Reply:
No, but I have no means to do it.